.github/scripts/verify-examples.pl

```diff
@@ -88,5 +97,8 @@ sub extract {
       $error |= testcompile($m);
       $error |= checksrc($m);
     }
+    else {
+      $error = 1;
```

Comment:
This will cause an error for the cases where the script does not extract any example, which does not have to be the same as examples lacking double-backslash... and if so, this is done with no message output so we can't tell which file this is. I suspect maybe the template?